README.md

```diff
@@ -29,6 +30,7 @@ region = "us-west-1"
 host = "Server1"
 whitelist_pass_through = False
 debug = False
+proxy_server_name = "http://myproxyserver.com:8080"
```

Comment:
I suggest also add proxy description into  collectd-cloudwatch/src/cloudwatch/config/plugin.conf, then we can easily know the configuration fields from the codes, I guess someone does not read readme carefully.